Makefile

```diff
@@ -43,7 +43,6 @@ static:
 xplatform-build:
 	GOOS=linux GOARCH=arm64 ./scripts/build true "" false
 	GOOS=windows GOARCH=amd64 ./scripts/build true "" false
-	GOOS=darwin GOARCH=amd64 ./scripts/build true "" false
```

Comment:
You can leave this here -- this only executes on linux and won't be affected by slowness.